enterprise/models.py

```diff
@@ -432,7 +432,7 @@ def catalog_contains_course(self, course_run_id):
             bool: Whether the enterprise catalog includes the given course run.
         """
         # Temporarily gate enterprise catalog api usage behind waffle sample
-        if utils.can_use_enterprise_catalog(self.uuid):
+        if utils.can_use_enterprise_catalog(str(self.uuid)):
```

Comment:
Would it make sense to cast the `uuid` as a string within `can_use_enterprise_catalog` so we don't have to do it in multiple places?